packages/orchestrator-ui-components/src/pages/metadata/WfoProductBlocksPage.tsx

```diff
@@ -245,9 +269,12 @@ export const WfoProductBlocksPage = () => {
                 localStorageKey={
                     METADATA_PRODUCT_BLOCKS_TABLE_LOCAL_STORAGE_KEY
                 }
-                onExportData={csvDownloadHandler(
+                onExportData={csvDownloadHandler<
+                    ProductBlocksResponse,
+                    ProductBlockDefinitionExportItem
```

Comment:
TS can infer these types, they are not needed here

Reply:
removed